common/changes/@microsoft.azure/openapi-validator-rulesets/rkmanda-RPC-Put-V1-29_2023-09-21-01-19.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Add RPC-Put-V1-29. The response schema of the 200 and 201 status codes of Put operation must be the same.",
```

Add the user-facing name:

```suggestion
      "comment": "Add ConsistentResponseSchemaForPut (RPC-Put-V1-29) rule. The response schema of the 200 and 201 status codes of Put operation must be the same.",
```